.gitignore

```diff
@@ -24,6 +24,7 @@ DatabaseManager.log*
 *.log
 local_settings.py
 db.sqlite3
+db.sqlite3.bak
```

Comment:
sometimes you'd want to experiment migrations on the _real_ database locally. When the migration is not reversible. It's reasonable to produce back ups this way.